app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/ReviewRegistrationController.scala

```diff
@@ -89,6 +76,15 @@ class ReviewRegistrationController @Inject() (
       registration.copy(metaData = updatedMetaData)
     }
 
+  private def markAsCannotYetStarted(
+    registration: Registration
+  )(implicit req: JourneyRequest[AnyContent]): Future[Either[ServiceError, Registration]] =
+    update { _ =>
+      val updatedMetaData =
+        registration.metaData.copy(registrationReviewed = false, registrationCompleted = false)
+      registration.copy(metaData = updatedMetaData)
+    }
+
```

Comment:
It makes it easier for the reader if private methods are below the methods which call them.

Reply:
done